src/TConsole.h

```diff
@@ -161,6 +161,8 @@ class TConsole : public QWidget
     void showEvent(QShowEvent* event) override;
     void hideEvent(QHideEvent* event) override;
     void setConsoleBgColor(int, int, int, int);
+    QColor getConsoleBgColor() const { return mBgColor; };
```

Comment:
This is why CodeFactor is unhappy with this PR...
```suggestion
    QColor getConsoleBgColor() const { return mBgColor; }
```
😁